src/context/products-context.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const productContext = React.createContext();
+const useProduct = () => React.useContext(productContext);
```

Comment:
you could directly destructure these two on line 1 and use them here, instead of having to explicitly mention React